sbin/cubeit

```diff
@@ -479,24 +492,20 @@ IFS=$OLDIFS
 
 	    # drop any properties and copy the containers to the installer
 	    for c in `strip_properties ${HDINSTALL_CONTAINERS}`; do
-		cp ${c} ${mnt_rootfs}/${INSTALLER_TARGET_IMAGES_DIR}/containers/
+		cp ${c} ${mnt_rootfs}/${INSTALLER_TARGET_IMAGES_DIR}/
```

Comment:
Why is this being changed ? These are supposed to be going into the containers subdir.